### PR TITLE
Small code improvements

### PR DESCRIPTION
**Pull request by jlogeart**

### CODE REVIEW

File: app/lib/Util.scala [hunk 1]
```diff
@@ -8,31 +8,16 @@ object Util {
...
   def removeKeys(
     data: Map[String, Seq[String]],
     keys: Set[String],
-  ): Map[String, Seq[String]] = {
-    data.filter { case (k, _) =>
-      !keys.contains(k)
-    }
-  }
+  ): Map[String, Seq[String]] =
+    data -- keys
```
Comment (benwaffle) [inline app/lib/Util.scala]: maybe just inline this?

File: app/lib/Util.scala [hunk 2]
```diff
@@ -8,31 +8,16 @@ object Util {
...
   def filterKeys(
     data: Map[String, Seq[String]],
     keys: Set[String]
-  ): Map[String, Seq[String]] = {
-    data.filter { case (k, _) =>
-      keys.contains(k)
-    }
-  }
+  ): Map[String, Seq[String]] =
+    data.filterKeys(keys)
```
Comment (benwaffle) [inline app/lib/Util.scala]: ditto

File: app/lib/Util.scala (hunk 1 above)
Comment (jlogeart) [reply app/lib/Util.scala]: As in `@inline` or as in removing the function?

Comment (jlogeart) [reply app/lib/Util.scala]: I vote for removing. Proceeding.

Comment (benwaffle) [reply app/lib/Util.scala]: removing

**Pull request merged**